src/eos-browser-appmode

```diff
@@ -60,14 +57,10 @@ if __name__ == '__main__':
    custom_uri = sys.argv[1]
    try:
       (wm_class, actual_uri) = parseURI(custom_uri)
-
-      subprocess.Popen(["gio", "open", actual_uri])
```

Comment:
This invocation doesn't check for errors, etc. There are variants which do, such as:

```python3
subprocess.run(..., check=True)
```

but at this point one may as well just exec.